docs/documentation/documentation_tips.md

```diff
@@ -0,0 +1,81 @@
+---
+id: documentation_tips
+title: Documentation Best Practices
+---
+
+This article describes the desired documentation standards in XSOAR content entities, and contains examples that can be very useful when writing documentation.
+
+## Entities Description Field
+### For playbook and scripts, use the following guideline:
+- Should start with the verb that describes what the entity does.
+- There's a limited space for descriptions, so we don't want to waste space (and time) with words that don't matter. 
+
+Bad example: The XYZ playbook is a playbook that...
+  We should shorten the description to what matters most.  
```

Comment:
i believe you want to add another new line here so it will be in a separate line, its currently in the same line:
![image](https://user-images.githubusercontent.com/31018228/105085269-2eb7d980-5aa0-11eb-9f0d-5ac40ccee8b4.png)

also, i would wrap the examples text in backticks or `>`